corgidrp/polflat.py

```diff
@@ -0,0 +1,377 @@
+import numpy as np
+from scipy import interpolate
+from scipy.ndimage import median_filter
+from scipy.ndimage import gaussian_filter as gauss
+from scipy import ndimage
+from scipy.signal import convolve2d
+import astropy.io.fits as fits
+from astropy.convolution import convolve_fft
+import photutils.centroids as centr
+from photutils.aperture import CircularAperture
+import corgidrp.mocks as mocks
+import corgidrp.data as data
+from astropy.stats import sigma_clipped_stats
+from photutils.detection import DAOStarFinder
+import corgidrp.flat as flat
+from skimage.transform import downscale_local_mean
+
+                                     
+def create_spatial_pol(dataset,nr=60,pfov_size=174,image_center_x=512,image_center_y=512,separation_diameter_arcsec=7.5,alignment_angle_WP1=0,alignment_angle_WP2=45,planet=None,band=None,prism='POL0'):
```

Comment:
what is the purpose of this function compared to the one that exists in mocks.py already? should we just delete it? if so, ignore all my comments on thsi function.